x-pack/legacy/plugins/alerting/server/lib/parse_duration.test.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { parseDuration } from './parse_duration';
+
+test('parses seconds', () => {
+  const result = parseDuration('10s');
+  expect(result).toEqual(10000);
+});
+
+test('parses minutes', () => {
+  const result = parseDuration('10m');
+  expect(result).toEqual(600000);
+});
+
+test('parses hours', () => {
+  const result = parseDuration('10h');
+  expect(result).toEqual(36000000);
+});
+
+test('parses days', () => {
+  const result = parseDuration('10d');
+  expect(result).toEqual(864000000);
+});
+
+test('throws error when the format is invalid', () => {
```

Comment:
should also add a test where there is not a unit suffix: `parseDuration('1000')`